src/controllers/groupe.controller.ts

```diff
@@ -0,0 +1,83 @@
+import { NextFunction, Request, Response } from 'express';
+import { Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import groupService from '@services/groups.service';
+
+class GroupController {
+  public groupService = new groupService();
```

Comment:
Consider passing objects through the constructor instead of initializing them internally. This practice helps maintain decoupling between components and makes testing easier by allowing dependencies to be injected.

```suggestion
  constructor(private groupService: GroupsService) {}
```